test/matchers/shared_examples/has_valid_rspec_message_methods.rb

```diff
@@ -7,6 +7,7 @@ module HasValidRspecMessageMethods
         case validator_sym
         when :aspect_ratio then matcher.rejecting(:square)
         when :attached then matcher
+        when :limit then matcher
```

Comment:
It should cause the matcher to return `false`, resulting in generating a Rspec failure message.
Reading your [test/dummy/app/models/limit/matcher.rb](https://github.com/igorkasyanchuk/active_storage_validations/pull/263/files#diff-b0a30e076bd2f69f5c85ff36251281bcc09d45601c23f120bf60e8b252da97e3), it could be `when :limit then matcher.max(6)` for example